packages/marko-web-omeda-identity-x/add-integration-hooks.js

```diff
@@ -29,6 +29,9 @@ module.exports = (params = {}) => {
     idxOmedaRapidIdentifyProp,
     omedaPromoCodeCookieName,
     omedaPromoCodeDefault,
+    onLoginLinkSentShouldAwait,
```

Comment:
Instead of making these individual props, can we move this into a single object called `shouldAwait`?

```js
shouldAwait: Joi.object({
  onLoginLinkSent: Joi.boolean().default(false),
  onAuthenticationSuccess: Joi.boolean().default(false),
  // ... and others
}).default(),
```

The just pass the corresponding value:
`shouldAwait: shouldAwait.onLoginLinkSent`